src/main/java/org/apache/accumulo/testing/performance/tests/ConditionalMutationsPT.java

```diff
@@ -138,12 +134,6 @@ public static double conditionalMutationsTime(ConditionalWriter cw, Long seq, St
 
     double rate = 10000 / ((t2 - t1) / 1000.0);
```

Comment:
nanoTime, rather than clock time, is more appropriate for differences. Could also convert the time explicitly using TimeUnit functions in the divisor rather than having a divisor that is itself a division problem.